langchain4j-core/src/main/java/dev/langchain4j/agent/tool/lambda/IdeaProxyLambdaMeta.java

```diff
@@ -0,0 +1,70 @@
+package dev.langchain4j.agent.tool.lambda;
+
+import java.lang.invoke.MethodType;
+import java.lang.reflect.Field;
+import java.lang.reflect.InvocationHandler;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.lang.reflect.Proxy;
+
+import static dev.langchain4j.agent.tool.lambda.LambdaMetaUtils.getInstantiatedMethod;
+import static dev.langchain4j.agent.tool.lambda.LambdaMetaUtils.setAccessible;
+
+/**
+ * Support for IDEA Evaluate Lambda expression
+ */
+class IdeaProxyLambdaMeta implements LambdaMeta {
```

Comment:
yep, it will support the IDEA Evaluate lambda expression.

The code does from https://github.com/baomidou/mybatis-plus/blob/3.0/mybatis-plus-core/src/main/java/com/baomidou/mybatisplus/core/toolkit/support/IdeaProxyLambdaMeta.java

But it is goal to `getter` or `setter` impl method name & Instantiated type. In there, it have to fetch the according method signature(parameter_type, parameter_name,return_type).

Reply:
## Normal ReflectLambdaMeta
![image](https://github.com/langchain4j/langchain4j/assets/30573693/19cde202-5957-4ee8-a27e-ea8574a33a92)

## IDEA Evaluate Expression IdeaProxyLambdaMeta
![image](https://github.com/langchain4j/langchain4j/assets/30573693/82dac841-2239-4d5a-a1e0-52fa506a5217)